inspire/modules/workflows/tasks/submission.py

```diff
@@ -86,11 +86,17 @@ def _create_ticket(obj, eng):
         rt_queue = cfg.get("CFG_BIBCATALOG_QUEUES") or queue
         sip = d.get_latest_sip(sealed=False)
         subject = u"Your suggestion to INSPIRE: {0}".format(d.title)
+        try:
+            user_comment = filter(lambda k: 'submitter' == k.get('source', ''),
+                                  sip.metadata.get("hidden_note", []))[0].get('extra_comments', None)
```

Comment:
I guess I meant `value`... But I believe I tested it like that, let me check it again!